### PR TITLE
Add Userscripts for Safari (iOS + macOS)

### PR DESCRIPTION
**Pull request by luckydonald**
https://apps.apple.com/app/userscripts/id1463298887

### CODE REVIEW

File: views/includes/documents/Userscripts-for-Safari.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Userscripts for Safari
+
+<img src="https://is1-ssl.mzstatic.com/image/thumb/Purple126/v4/60/80/51/60805145-b1d7-ada7-774e-5baeb7055b15/AppIcon-85-220-4-2x.png/1024x0w.webp" width="128" height="128" align="right">
```
Comment (Martii) [inline views/includes/documents/Userscripts-for-Safari.md:3]: Any chance at getting the SVG of this?

File: views/includes/documents/Userscripts-for-Safari.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+To get userscripts going in Userscripts, first you have to install it from the [Apple Appstore][appstoreWebsite]. Install it like any other App.
+
+### Installing Userscripts
+
+Once Userscripts is installed, installing userscripts from [OpenUserJS.org][oujs] is simple. Navigate to the OpenUserJS page for the script, then click the blue "Install" button at the top of the page.
+You will be directed to a page showing the source code of that script.
+Open the Userscripts extension, and click where it says <kbd>Userscript Detected: <u>Click To Install</u></kbd>.
+
+<img width="413" alt="image" src="https://github.com/luckydonald-forks/OpenUserJS.org/assets/2737108/cb2565cb-e708-4e58-89d3-2e669dc9fabd">
```
Comment (Martii) [inline views/includes/documents/Userscripts-for-Safari.md:17]: This will have to change to something repo local eventually... but good example for now.

**Comment by Martii** [comment]
Overall it looks good... but the SVG is something that is rather important for presentation. I'll still have to try it on macOS and borrow a friends iPhone for a test at some point. I do appreciate the code styling match.

However... this is my personal fork of upstream... so anything done here will have to be done on OpenUserJs.org as a PR there.

File: views/includes/documents/Userscripts-for-Safari.md (hunk 1 above)
Comment (luckydonald) [reply views/includes/documents/Userscripts-for-Safari.md:3]: Not that I know of any. Probably should be mirrored locally here anyway.

**Comment by luckydonald** [comment]
> However... this is my personal fork of upstream... so anything done here will have to be done on OpenUserJs.org as a PR there.

Oh. I'm not sure how that happened. I think a google search led me here.

Still thanks for the feedback in that case.

**Comment by Martii** [comment]
@luckydonald 

I'd appreciate it if you would redo this [upstream](https://github.com/OpenUserJS/OpenUserJS.org).

I'm going to close this here.

Thanks.

**Comment by Martii** [comment]
Just a random followup... tried it on an iPhone 6S and doesn't appear to work... script install button on OUJS is translated to a text file in Safari. That's disappointing.

**Comment by luckydonald** [comment]
For your followup:
Just open the Userscript plugin from the safari plugin menu.
It will ask you to install the script.
Did just test this with the OUJS page, it works.

**Pull request closed without merging**